docs/running-workflows/launching-workflows/index.md

```diff
@@ -0,0 +1,125 @@
+
+- `https://zenodo.org/record/5752285/files/circular-health-data-processing-master.zip` (ZIP file  with necessary files to run workflow)
+- `https://example.org/reana.yaml` (REANA specification file)
+
+### name (optional)
+
+The launch page accepts an optional `name` argument that will determine the desired name of the launched workflow.
+If the argument is missing, REANA will determine an appropriate name based on the URL of the external source,
+for example, the repository name and branch. Example usage:
+
+```text
+https://reana.cern.ch/launch?name=my-new-name&url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+### specification (optional)
+
+If REANA specification file is different from the default `reana.yaml`,
+you can specify the path to the desired specification file using `specification` argument:
+
+```text
+https://reana.cern.ch/launch?specification=reana-snakemake.yaml?url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+### parameters (optional)
+
+REANA specification file allows defining parameters for the workflow.
+In some cases, you may want to change the default values of these parameters.
+
+For example, [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit)
+has `events` parameter in the specification file. By default, it is `20000`. Let's reduce this parameter to `10000`:
+
+```text
+https://reana.cern.ch/launch?parameters={"events": "10000"}&url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+## Launcher badges
+
+In order to facilitate launching your analysis workflows directly from GitHub or GitLab, you may want to generate a badge pointing to the launcher URL with the desired arguments and then include this badge in your analysis documentation.
+
+The standard badge looks like this:
+
+![Launch on REANA](https://www.reana.io/static/img/badges/launch-on-reana.svg)
+![Launch on REANA at CERN](https://www.reana.io/static/img/badges/launch-on-reana-at-cern.svg)
+![Launch with CWL on REANA at CERN](https://www.reana.io/static/img/badges/launch-with-cwl-on-reana-at-cern.svg)
+![Launch with Yadage on REANA at CERN](https://www.reana.io/static/img/badges/launch-with-yadage-on-reana-at-cern.svg)
+![Launch with Serial on REANA at CERN](https://www.reana.io/static/img/badges/launch-with-serial-on-reana-at-cern.svg)
+![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)
+
+### Markdown
+
+Badge can be added in Markdown using the following syntax:
+
+```markdown
+[![alt text](https://www.reana.io/static/img/badges/launch-on-reana.svg)](<WORKFLOW_LAUNCH_URL>)
+```
+
+Substitute your URL instead of `WORKFLOW_LAUNCH_URL`:
+
+```markdown
+[![Launch RooFit demo workflow from GitHub on REANA](https://www.reana.io/static/img/badges/launch-on-reana.svg)](https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit)
+```
+
+### HTML
+
+Badge can be added in HTML using the following syntax:
```

Comment:
You can include one of the above badges into your HTML documentation pages using the following syntax: